pkg/webui/components/icon/index.js

```diff
@@ -38,7 +38,7 @@ const hardcoded = {
   general_settings: 'settings',
   location: 'place',
   user: 'person',
-  event: 'offline_bolt',
+  event: 'wifi',
```

Comment:
I think we still need to work on the event icons and likely rely on another icon set than material icons. Albeit good enough for now.